src/api/v1/public/validation/schemas/apps/helpers/dataObj.js

```diff
@@ -7,8 +7,12 @@ const dataObj = Joi.object()
 .pattern(
     Joi.string().max(100),                                  // Keys
     Joi.alternatives().try(                                 // Values
-        Joi.string().max(500),
-        Joi.number().max(999999999999999999999999999999),   // 30 digit
+        Joi.string()
+           .min(1)
+           .max(500),
+        Joi.number()
+           .min(-999999999999999999999999999999)            // 30 digit
```

Comment:
The goal is to limit this to 30 digit numbers. This means having a lower limit, in addition to an upper limit.